### ISSUE TITLE
[ FEATURE REQUEST ] - Natsuki Bedroom Background

### ISSUE THREAD

**Issue by nyxedmisfit** [Just-Natsuki-Team/NatsukiModDev#288]
# Overview:
[brief description of what this aims to achieve]
a background/room feature where natsuki is in her bedroom.
## Must haves:
[features that you deem essential for this issue]
A more comfortable environment for our baby girl suki.
## Could haves:
[features that you deem would be good, but not essential, for this issue]
We could open the game to natsuki reading on her bed, or to her cleaning up. 
## Is this feature request related to a problem? Please describe.
[A clear and concise description of what the problem is. Ex. I'm always frustrated when...]
The regular background is rather dull.

**Comment by Blizzardsev** [comment]
Hi @nyxedmisfit, thanks for your suggestion.

This is an interesting idea; we're definitely open to the idea of allowing the player and Natsuki to spend their time in a few more places.

Beyond a background change, did you have any thoughts as to how to make the bedroom differ from the classroom? For example, would it unlock any topics? If so, what sorts of conversations? Etc.

**Comment by nyxedmisfit** [comment]
@Blizzardsev I feel like it would unlock conversations about dreams, you could prompt natsuki to take a rest. You could ask about things in her room, prompting her to bring up memories. Like maybe she has a trophy for a baking competition, she could tell you about what she baked, the other people there. Maybe she would have stuffed animals, you could ask her about them. I'm still thinking of things and I plan on using the sims to design a room as an example (just so i can think of more things.)

**Comment by kaydenkimkim** [comment]
i agree i feel like its kind of sad always seeing her in the classroom a bedroom would be perfect

**Comment by RZed01** [comment]
Not to mention that BG for Natsuki's bedroom already exists, if I remember correctly.
